improver/utilities/cube_manipulation.py

```diff
@@ -550,7 +550,7 @@ def clip_cube_data(cube: Cube, minimum_value: float, maximum_value: float) -> Cu
     original_methods = cube.cell_methods
 
     result = iris.cube.CubeList()
-    for cube_slice in cube.slices([cube.coord(axis="y"), cube.coord(axis="x")]):
+    for cube_slice in cube.slices(cube.coord(axis="x")):
```

Comment:
I think the only other place this is used is in `‎improver/calibration/simple_bias_correction.py`. Is changing to only slice over the x axis likely to dramatically change performance in the CLI using this plugin? I don't know if we use this CLI in our IMPROVER suite, but I think the bias correction plugin and CLI were developed by BoM so I assume they might use it

Reply:
I think you're right that this change would possibly effect the bias correct which I hadn't considered. Given this and the fact that the clip_cube_data seems to have been set up as a way to clip data using very little memory I have returned the functon to how it was previously. In the clip cli I now just call the np.clip for all cubes. I checked with the memory profiler and the maximum memory use when using a full cube of gridded data was 398.41MiB which seems to be acceptable.